sphinx_gallery/gen_rst.py

```diff
@@ -505,16 +530,16 @@ def execute_script(code_block, example_globals, image_path, fig_count,
         fig_count += 1  # raise count to avoid overwriting image
 
         # Breaks build on first example error
-
-        if gallery_conf['abort_on_example_error']:
+        # Eventually this should hopefully be unified with the default conf
+        if gallery_conf.get('abort_on_example_error', True):
```

Comment:
OK something to look at in the future. It'd be great if you could provide something that reproduces the problem.